set.mm

```diff
@@ -632529,6 +632529,116 @@ fixed reference functional determined by this vector (corresponding to
       $.
   $}
 
+  ${
+    $d A k x $.  $d A q r x $.  $d B k x $.  $d F q r $.  $d G q r x $.
+    $d H q r $.  $d L q r $.  $d U q r $.  $d k ph x $.  $d ph q x $.
+    3factsumint1.1 $e |- A = ( L [,] U ) $.
+    3factsumint1.2 $e |- ( ph -> B e. Fin ) $.
+    3factsumint1.3 $e |- ( ph -> L e. RR ) $.
+    3factsumint1.4 $e |- ( ph -> U e. RR ) $.
+    3factsumint1.5 $e |- ( ( ph /\ x e. A ) -> F e. CC ) $.
+    3factsumint1.6 $e |- ( ph -> ( x e. A |-> F ) e. ( A -cn-> CC ) ) $.
```

Comment:
I believe the first hypothesis:
```
|- ( ( ph /\ x e. A ) -> F e. CC )
```
Can be derived from the second:
```
|- ( ph -> ( x e. A |-> F ) e. ( A -cn-> CC ) )
```
See for example ~ cncff

Reply:
Ah, well time to fix it.

Reply:
I would have fixed them from the last to the first, so that not everything is broken.

For fixing a theorem, just reopen the proof file, remove the "first" hypothesis, and fill the blank.
This should work:
```
h10::         |- ( ph -> ( x e. A |-> F ) e. ( A -cn-> CC ) )
20::cncff            |- ( ( x e. A |-> F ) e. ( A -cn-> CC ) -> ( x e. A |-> F ) : A --> CC )
30:10,20:syl        |- ( ph -> ( x e. A |-> F ) : A --> CC )
40:30:fvmptelrn   |- ( ( ph /\ x e. A ) -> F e. CC )
```